weather.go

```diff
@@ -1,5 +1,138 @@
 package weather
 
-func Get(location, token string) ([]byte, error) {
-	return nil, nil
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"math"
+	"net/http"
+	"os"
+	"strings"
+)
+
+type Weather struct {
+	Name    string
+	Weather []struct {
+		ID          int
+		Main        string
+		Description string
+		Icon        string
+	}
+	Main struct {
+		Temp      float64
+		FeelsLike float64
+		TempMin   float64
+		TempMax   float64
+		Humidity  int
+	}
+}
+
+type CityUnknown struct {
+	Cod     string
+	Message string
+}
+
+type Conditions struct {
+	City    string
+	OneWord string
+	Celsius float64
+}
+
+func CliOutput(token string, args []string) (output string) {
+
+	location := LocationFromArgs(args)
+	url := BuildURL(token, location)
+	callURL := CallURL(url)
+	conditions, err := Get(callURL)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	output = fmt.Sprintf("city: %s\nweather: %s\nCelsius: %v\n", conditions.City, conditions.OneWord, conditions.Celsius)
```

Comment:
This might be better as a `String` method on the `Conditions` type. Then you can simply write:

```go
return conditions.String()
```

Or, even better:

```go
fmt.Println(conditions)
```